Tests/RoslynPluginGeneratorTests/AnalyzerPluginGeneratorTests.cs

```diff
@@ -75,15 +76,86 @@ public void Generate_LicenseAcceptanceNotRequired_Succeeds()
             logger.AssertWarningNotLogged("child.id");
             logger.AssertWarningNotLogged("grandchild.id");
 
-            // 2. Acceptance not required -> succeeds if accept = true
-            args = CreateArgs("parent.id", "1.0", "cs", null, true /* accept licenses */ , outputDir);
+            // 1. b) Acceptance not required -> succeeds if accept = false, generate dependencies = true
+            args = CreateArgs("parent.id", "1.0", "cs", null, false /* accept licenses */ ,
+                true /* generate plugins for dependencies */, outputDir);
             result = apg.Generate(args);
             Assert.IsTrue(result, "Generator should succeed if there are no licenses to accept");
 
             logger.AssertErrorsLogged(0);
             logger.AssertWarningNotLogged("parent.id"); // not expecting warnings about packages that don't require acceptance
             logger.AssertWarningNotLogged("child.id");
             logger.AssertWarningNotLogged("grandchild.id");
+
+            // 2. a) Acceptance not required -> succeeds if accept = true, generate dependencies = false
+            args = CreateArgs("parent.id", "1.0", "cs", null, true /* accept licenses */ , 
+                false, outputDir);
+            result = apg.Generate(args);
+            Assert.IsTrue(result, "Generator should succeed if there are no licenses to accept");
+
+            logger.AssertErrorsLogged(0);
+            logger.AssertWarningNotLogged("parent.id"); // not expecting warnings about packages that don't require acceptance
+            logger.AssertWarningNotLogged("child.id");
+            logger.AssertWarningNotLogged("grandchild.id");
+
+            // 2. b) Acceptance not required -> succeeds if accept = true, generate dependencies = true
+            args = CreateArgs("parent.id", "1.0", "cs", null, true /* accept licenses */ ,
+                true /* generate plugins for dependencies */, outputDir);
+            result = apg.Generate(args);
+            Assert.IsTrue(result, "Generator should succeed if there are no licenses to accept");
+
+            logger.AssertErrorsLogged(0);
+            logger.AssertWarningNotLogged("parent.id"); // not expecting warnings about packages that don't require acceptance
+            logger.AssertWarningNotLogged("child.id");
+            logger.AssertWarningNotLogged("grandchild.id");
+        }
+
+        [TestMethod]
+        public void Generate_LicenseAcceptanceNotRequired_NoAnalyzersInTarget()
```

Comment:
New test case. Most of the other test cases are just "as previous, but also with the recursive flag on".